src/mrack/outputs/ansible_inventory.py

```diff
@@ -143,7 +144,7 @@ def create_ansible_host(self, name):
             "ansible_host": ansible_host,
             "ansible_python_interpreter": python,
             "ansible_user": ansible_user,
-            "meta_fqdn": f"{get_shortname(name)}.{dom_name}",
+            "meta_fqdn": f"{get_fqdn(name, dom_name)}",
```

Comment:
It must be ensured that `meta_fqdn` = `meta_hostname` + `meta_domain`.

Reply:
The intent here is to be able to override this assumption and have hosts from subdomains directly in the same tld domain by specifying names as fqdn.
```
domains:
  - name: test
    type: sssd
    hosts:
      - name: client.test
        group: medium
        role: client
        os: rhel-9.2
      - name: dns.test
        group: xsmall
        role: dns
        os: rhel-9.2
      - name: master.ldap.test
        group: medium
        role: ldap
        os: rhel-9.2
      - name: master.ipa.test
        group: medium
        role: ipa
        os: rhel-9.2
```

Reply:
i would expect domain name to be: something.com

Reply:
To keep discussion on one place, let's continue the discussion in the issue: https://github.com/neoave/mrack/issues/237

Reply:
Resolved.